analyzers/gradle/gradle.go

```diff
@@ -35,6 +35,8 @@ type Options struct {
 	Online            bool   `mapstructure:"online"`
 	AllSubmodules     bool   `mapstructure:"all-submodules"`
 	AllConfigurations bool   `mapstructure:"all-configurations"`
+	Timeout           int    `mapstructure:"timeout"`
```

Comment:
Why not let the user supply a duration string such as "5m30s" and parse it using [ParseDuration](https://godoc.org/time#ParseDuration)? This could make the UX a little nicer when otherwise people would need to convert their desired timeout to seconds. But then we'd need to document how a duration is specified—but we should document this configuration anyway. What do you think?

This would probably suffice to explain it:
> A duration string is a sequence of numbers, each with a unit suffix, such as "500ms", "1.5m", or "2h45m". Valid time units are "ms", "s", "m", "h".

Reply:
This is a really good thought and idea with the command ParseDuration. I'm a little conflicted with using this in practice because it adds additional complexity for the user. I'm inclined to disagree with using this for the sake of simplicity and because timeouts should always be set to low values. This is really interesting though and I'm going to play around with it before I make up my mind.

Reply:
I changed it to this, I think this is a little more future proofed. Good idea.